web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -28,6 +28,8 @@ import { UserPreferences } from 'gen-proto-ts/teleport/userpreferences/v1/userpr
 
 import { Resource } from 'gen-proto-ts/teleport/userpreferences/v1/onboard_pb';
 
+import { ExternalLinkIcon } from 'design/SVGIcon';
```

Comment:
We need to ask @roraback to export this Icon so we can use the `design/Icon` version instead (these are old icons... long story)

Reply:
It's already exported. It's the "NewTab" icon under the standard "Icon" component in Storybook. Nothing under the "Custom Icons" (aka "SVGIcon") component should be used—it's meant to be deprecated, but its death has been excruciatingly slow.
![image](https://github.com/user-attachments/assets/062937b0-c5f5-4d10-be0c-d650ebaf2288)

Reply:
Ah - is that a blocker, or something to come back to? I don't see another version of this icon under `design/Icon`...

Reply:
Since it is already exported, its just a simple change
```suggestion
import { NewTab } from 'design/Icon';
```

and you may need to update the size at the callsite

Reply:
I think this gets it to match your screenshot
```js
 <NewTab color="text.muted" size={18} />
```

Reply:
Got it, thx!